tests/test_op13_schema_traits_validation.py

```
@@ -0,0 +1,1788 @@
+            },
+            "register derived missing topicRef from $ref trait",
+        ),
+        _validate_schema(
+            (
+                "gts.x.test13.refm.event.v1~"
+                "x.test13._.ref_incomplete.v1~"
+            ),
+            False,
+            "validate should fail - topicRef not resolved",
+        ),
+    ]
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)